documentation/book/proc-deploying-the-topic-operator-standalone.adoc

```diff
@@ -0,0 +1,63 @@
+
+[id='deploying-the-topic-operator-standalone']
+= Deploying the Topic Operator standalone
+
+Deploying the Topic Operator as a standalone component is more complicated than installing it using the Cluster Operator, but is more flexible.
+For instance is can operate _with_ any Kafka cluster, not necessarily one deployed by the Cluster Operator.
+
+.Prerequisites
+
+* An existing Kafka cluster for the Topic Operator to connect to.
+
+.Procedure
+
+. Edit the `examples/install/cluster-operator/05-Deployment-strimzi-topic-operator.yaml` resource. You will need to change the following
++
+* The `STRIMZI_KAFKA_BOOTSTRAP_SERVERS` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to a list of bootstrap brokers in your Kafka cluster, given as a comma-separated list of `_<hostname>_:‍_<port>_` pairs.
+* The `STRIMZI_ZOOKEEPER_CONNECT` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to a list of the Zookeeper nodes, given as a comma-separated list of `_<hostname>_:‍_<port>_` pairs. This should be the same Zookeeper cluster that your Kafka cluster is using.
+* The `STRIMZI_NAMESPACE` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to the {ProductPlatformName} namespace in which you want the operator to watch for  `KafkaTopic` resources.
+
+. Deploy the Cluster Operator.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
++
+[source,shell]
+kubectl apply -f examples/install/cluster-operator
++
```

Comment:
You want to close this 'ifdef' before the OpenShift info.

Reply:
This is intentional, actually, but @scholzj didn't like it much either. The full thing looks like this

```
ifdef::Kubernetes[]
On {KubernetesName} this can be done using `kubectl apply`:
+
[source,shell]
kubectl apply -f examples/install/cluster-operator
+
On {OpenShiftName} this can be done using `oc apply`:
endif::Kubernetes[]
ifndef::Kubernetes[]
This can be done using `oc apply`:
endif::Kubernetes[]
+
[source,shell]
oc apply -f examples/install/cluster-operator
```

So when both platforms are being doc'd you get "On K8s this can be done by ... On OS this can be done by ..." which reads very nicely. When just OS is being documented it would be rubbish if it read "On OS this can be done ..." because _obviously it's on OS_. Imho it reads much better to just say "This can be done <example>". But clearly I'm in the minority here, so I will change it.